portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -164,10 +164,25 @@ definition {
 	}
 
 	macro editText {
-		Type(
-			key_fieldName = "${fieldName}",
-			locator1 = "FormFields#TEXT_FIELD",
-			value1 = "${fieldValue}");
+		var browserType = PropsUtil.get("browser.type");
+
+		if ("${browserType}" == "internetexplorer") {
+			Type(
+				key_fieldName = "${fieldName}",
+				locator1 = "FormFields#TEXT_FIELD",
+				value1 = "");
+
```

Comment:
is this necessary?

Reply:
Yes, because sendkeys doesn't remove the default text that already exist in field. But type function can remove first, then type the new text. So I update macro to type then sendkeys, hopefully can fix this issue.